components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -516,18 +515,17 @@ public void cleanUpApplicationRegistrationByApplicationId(int applicationId, Str
     }
 
     /**
-     *
-     * @param jsonString this string will contain oAuth app details
-     * @param userName user name of logged in user.
-     * @param clientId this is the consumer key of oAuthApplication
+     * @param jsonString      this string will contain oAuth app details
+     * @param userName        user name of logged in user.
+     * @param clientId        this is the consumer key of oAuthApplication
      * @param applicationName this is the APIM appication name.
      * @param keyType
-     * @param tokenType this is theApplication Token Type. This can be either default or jwt.
-     * @param keyManagerName key Manager name
+     * @param tokenType       this is theApplication Token Type. This can be either default or jwt.
+     * @param keyManagerName  key Manager name
      * @return
      * @throws APIManagementException
      */
-    @Override
+    @Override //TODO: (VirajSalaka) See usages too
     public Map<String, Object> mapExistingOAuthClient(String jsonString, String userName, String clientId,
```

Comment:
I think this is an older method where we pass username and appname to identify app instead of passing app uuid. Shall we rewrite this method to take the uuid please?